src/chain/shared_state.rs

```diff
@@ -34,6 +34,8 @@ pub struct SharedState {
     /// The latest few fully signed infos of our own sections.
     /// This is not a `BTreeSet` as it is ordered according to the sequence of pushes into it.
     pub our_infos: NonEmptyList<SectionInfo>,
+    /// Info about all members of our section - elders, adults and infants.
+    pub our_members: BTreeMap<PublicId, MemberInfo>,
```

Comment:
It seem we've added dead code and an empty collection.

Would it make sense to populate it and use it as part of this PR. i.e:
- When a node drop put it in the `Left` state.
- Currently we only have elder, but getting consensus on it with the NetworkEvent::SectionInfo(....).